apps/wallet/src/background/account-sources/AccountSource.ts

```diff
@@ -46,6 +46,10 @@ export abstract class AccountSource<
 	protected clearEphemeralValue() {
 		return clearEphemeralValue(this.id);
 	}
+
+	public async setNickname(nickname: string) {
```

Comment:
we should probably remove this. This is the `AccountSource` which we don't need a nickname (for now). Also it seems the function updates the account and not the `AccountSource`